README.md

```diff
@@ -1,5 +1,6 @@
-[![R-CMD-check](https://github.com/sidora-tools/sidora.core/actions/workflows/check-release.yaml/badge.svg)](https://github.com/sidora-tools/sidora.core/actions/workflows/check-release.yaml)
-[![Coverage Status](https://img.shields.io/codecov/c/github/sidora-tools/sidora.core/master.svg)](https://codecov.io/github/sidora-tools/sidora.core?branch=master)
+[![R-CMD-check](https://github.com/sidora-tools/sidora.core/actions/workflows/check-release.yaml/badge.svg?style=for-the-badge&logo=appveyor)](https://github.com/sidora-tools/sidora.core/actions/workflows/check-release.yaml)
+[![Coverage Status](https://img.shields.io/codecov/c/github/sidora-tools/sidora.core/master.svg?style=for-the-badge&logo=appveyor)](https://codecov.io/github/sidora-tools/sidora.core?branch=master)
+[![Latest release](https://img.shields.io/badge/Version-1.0.0-blue)
```

Comment:
There is a special shields badge for the current R package version: ![GitHub R package version](https://img.shields.io/github/r-package/v/sidora-tools/sidora.core)

Reply:
```
![GitHub R package version](https://img.shields.io/github/r-package/v/sidora-tools/sidora.core)
```

Reply:
I thought that referred to the version of R the package supports?

Reply:
At least that's how I would understand it...

Reply:
Nope - it magics the package version out of the DESCRIPTION file